istioctl/cmd/proxyconfig.go

```diff
@@ -847,7 +868,11 @@ func logCmd() *cobra.Command {
 				if err != nil {
 					return err
 				}
-				if isZtunnelPod(pod, podNamespace) {
+				ztunnelPod, err := isZtunnelPod(pod, podNamespace)
+				if err != nil {
+					return fmt.Errorf("failed to create k8s client: %v", err)
```

Comment:
nit: the error message here is too specific; if there were other possible errors in isZtunnelPod, we wouldn't want to make the top-level message specific to the kubeClient.